bot/controllers/verify.js

```diff
@@ -0,0 +1,152 @@
+const models = require('../../db/models');
+const uuidv4 = require('uuid/v4');
+const util = require('apex-util');
+const nodemailer = require('nodemailer');
+require('dotenv').config();
+
+module.exports = () => {
+  const _run = (message) => {
+    const ctrls = [{
+      cmd: '!verify',
+      example: '!verify <email_address>',
+      title: 'Verify Email Address',
+      desc: 'Verify user\'s email address',
+      showWithHelp: true,
+      posTargetUser: null,
+      posSecondaryCmd: null,
+      regexSplitCharacter: null,
+      allowInDM: false,
+      resType: 'reply',
+      action: (message, ctrl, msg) => {
+        const validDomains = ['student.fullsail.edu', 'fullsail.edu'];
+        const email = msg.parsed[1].toLowerCase();
+        const emailDomain = email.split('@').pop();
+        // Method to generate random numeric verification code
+        // Modified to fit style guide from this SO answer:
+        // https://stackoverflow.com/a/39774334
+        const generateCode = (n) => {
```

Comment:
A utility function like this should be reusable and defined outside of the scope of this ctrl

Reply:
Fixed!

Reply:
Thank you, the utility function is now reusable and resides outside of the ctrl's scope.